schema-source/RSxxGlzSys.schema.yaml

```diff
@@ -0,0 +1,261 @@
+      Data Type: "({RSxxSub}, {FutureRSxxFilm}, {Cavity})"
+      Required: True
+    level_type:
+      Description: "Indicates the type of layer being referenced"
+      Data Type: "<LevelOptions>"
+      Required: True
+      Notes: "is this even needed or is their a way to determine this?"
+
+LevelOptions:
+  Object Type: "Enumeration"
+  Enumerators:
+    GLAZING_SUBSTRATE:
+      Display Text: "Glazing substrate"
+    FILM_OR_COATING:
+      Display Text: "Film or coating"
+    CAVITY:
+      Display Text: "Cavity"
+    OTHER:
+      Display Text: "other"
+
+FutureRSxxFilm:
+  Object Type: "Data Group"
+  Data Elements:
+    name:
+      Description: "name"
+      Data Type: "String"
+      Notes: "this is just a stand in for now. It will be very similar to RSxxSub but without angle sensitivity."
+
+Cavity:
+  Object Type: "Data Group"
+  Data Elements:
+    thickness:
+      Description: "The total thickness of the cavity."
+      Data Type: "Numeric"
+      Units: "m"
+      Required: True
+    fill:
+      Description: "Fill"
+      Data Type: "<FillOptions>"
+      Required: True
+    between_glass_attachment:
+      Description: "Window attachment that located in the cavity"
+      Data Type: "{FutureRSxxAttach}"
+      Notes: "This is referencing an RS that has not yet been developed but will represent blinds, shades, etc. that are located between the panes of glass in the cavity."
+    conductivity:
+      Description: "Conductivity"
+      Data Type: "Numeric"
+      Units: "W/m-K"
+      Constraints: ">=0"
+      Required: if fill=OTHER
+    viscosity:
+      Description: "Viscosity"
+      Data Type: "Numeric"
+      Units: "kg/m-s"
+      Constraints: ">=0"
+      Required: if fill=OTHER
+    specific_heat:
+      Description: "Specific heat"
+      Data Type: "Numeric"
+      Units: "J/kg-K"
+      Constraints: ">=0"
+      Required: if fill=OTHER
+    density:
+      Description: "Density"
+      Data Type: "Numeric"
+      Units: "kg/m3"
+      Constraints: ">=0"
+      Required: if fill=OTHER
+    prandtl_number:
+      Description: "Prandtl_Number"
+      Data Type: "Numeric"
+      Required: if fill=OTHER
+    molecular_weight:
+      Description: "Molecular weight"
+      Data Type: "Numeric"
+      Units: "g/mol"
+      Constraints: ">=0"
+      Required: if fill=OTHER
+
+FillOptions:
+  Object Type: "Enumeration"
+  Enumerators:
+    AIR:  
+      Display Text: "Air"
+    ARGON:
+      Display Text: "Argon"
+    KRYPTON:
+      Display Text: "Krypton"
+    XENON:
+      Display Text: "Xenon"
+    VACUUM:  
+      Display Text: "Vacuum"
+    OTHER:
+      Display Text: "Other"
+
+FutureRSxxAttach:
+  Object Type: "Data Group"
+  Data Elements:
+    name:
+      Description: "name"
+      Data Type: "String"
+      Notes: "This is just a placeholder. This is referencing an RS that has not yet been developed but will represent blinds, shades, etc."
+
+ColorTransmittance:
+  Object Type: "Data Group"
+  Data Elements:
```

Comment:
It makes a difference whether the irradiance is isotropic or from a specific direction. It makes also a difference from which direction, especially when shading is included.

Reply:
I didn't realize this had an angular dependency, thanks.